### ISSUE TITLE
Wanna ask the spec of the computer 

### ISSUE THREAD

**Issue by s10627070** [WeijingShi/Point-GNN#88]
Hi, Weijiing
When I tried to train the point-GNN model, the terminal showed me the error of OOM, may I ask you about the spec of your computer when you were training the Point-GNN ?

BTW thank you for developing such a great GNN model.

**Comment by WeijingShi** [comment]
Hi @s10627070,  I mainly used two 1080ti or two 2080ti for the training. Occasionally, it still went OOM. Kill it and restart, the code will continue from the previous checkpoint.  It may be better to use 4 GPUs if available. Thanks,

**Comment by s10627070** [comment]
Is there any way I can use only 1 GPU (RTX 3050 Laptop) to run the code?
Becuz I just wanna try this program if it is able to run.
Like deleting most of the datasets ?

**Comment by WeijingShi** [comment]
For the inference, tensorflow can run on CPU as well. All the needed is to
hide your GPU by ```CUDA_VISIBLE_DEVICES= python run.py ...```.
The training code though assigns batch to devices explicitly. If a lower
accuracy is not an issue, you can downsample the point cloud a lot to
reduce the memory requirements.
The downsampling can be done by increasing the ```base_voxel_size```
in  ```_config```
files or use ```scripts/point_cloud_downsample.py``` to downsample the
KITTI dataset directly.

Hope it helps,
Weijing
